packages/design-system/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@repo/design-system",
+  "version": "0.0.0",
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "types": "./index.ts"
+    }
+  },
+  "files": [
+    "dist"
+  ],
+  "type": "module",
+  "license": "MIT",
+  "scripts": {
+    "build": "run-p build:*",
+    "build:lib": "vite build --config ./vite.config.ts"
```

Comment:
No need to use the `--config` flag here as you've only go the one standard named config. It will get picked up automatically.